team-norms/retrospectives.md

```diff
@@ -0,0 +1,66 @@
+# Retrospectives
+
+These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose. 
```

Comment:
Given that retrospectives are to reflect on work done, it feels appropriate to do them at the end of a clean slate.

What do you propose?